### ISSUE TITLE
Errors experienced when enabling texture packs

### ISSUE THREAD

**Issue by src4026** [minetest/minetest#13835]
### Minetest version

```true
Minetest 5.8.0-dev-c247761 (Linux)
Using Irrlicht 1.9.0mt12
Using LuaJIT 2.1.0-beta3
BUILD_TYPE=Release
RUN_IN_PLACE=1
USE_CURL=1
USE_GETTEXT=1
USE_SOUND=1
STATIC_SHAREDIR="."
STATIC_LOCALEDIR="locale"
```

### Operating system and version

Pop!_OS 22.04

### Summary

I cloned the latest version of the minetest `master` branch right now and installed a few texturepacks. Upon going to the content tab and double clicking and/or clicking the `User texturepack` button, it pops up an error but in the end *does* select it and shows the following:
```
Runtime error from mod '' in callback handleMainMenuButtons(): /home/sasharc/minetest/bin/../builtin/common/filterlist.lua:209: attempt to index local 'self' (a nil value)
stack traceback:
	/home/sasharc/minetest/bin/../builtin/common/filterlist.lua:209: in function 'size'
	...sasharc/minetest/bin/../builtin/mainmenu/tab_content.lua:69: in function 'get_formspec'
	/home/sasharc/minetest/bin/../builtin/fstk/tabview.lua:67: in function 'get_formspec'
	/home/sasharc/minetest/bin/../builtin/fstk/ui.lua:101: in function 'update'
	/home/sasharc/minetest/bin/../builtin/fstk/ui.lua:145: in function 'handle_buttons'
	/home/sasharc/minetest/bin/../builtin/fstk/ui.lua:186: in function </home/sasharc/minetest/bin/../builtin/fstk/ui.lua:172>
```

### Steps to reproduce

- Install a texture pack
- Double click to enable it
- Errors are shown (but in the end texturepack is selected)

**Comment by grorp** [comment]
Will be fixed by #13829.

**Comment by src4026** [comment]
> Will be fixed by #13829.

Ah, great, there's already a PR for it. Thanks! 👍
